udata_front/theme/gouvfr/datagouv-components/src/api/api.ts

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
```

Comment:
It mostly seem like duplicate contents with assets/js/plugins/api.ts. Can't we import the main assets from datagouv-components?